isort/settings.py

```diff
@@ -45,9 +45,12 @@
 from .wrap_modes import from_string as wrap_mode_from_string
 
 if TYPE_CHECKING:
-    tomli: Any
+    tomllib: Any
 else:
-    from ._vendored import tomli
+    if sys.version_info >= (3, 11):
+        import tomllib
+    else:
+        from ._vendored import tomli
```

Comment:
```suggestion
        from ._vendored import tomli as tomllib
```

this will not work.. you will probably need this to keep working with older versions